test/OpenTelemetry.Exporter.Prometheus.Tests/PrometheusExporterExtensionsTests.cs

```diff
@@ -27,30 +28,40 @@ namespace OpenTelemetry.Exporter.Prometheus.Tests
 {
     public sealed class PrometheusExporterExtensionsTests
     {
-        [Fact]
-        public void WriteMetricsCollectionTest()
+        [Theory]
+        [InlineData(nameof(WriteLongSum))]
+        [InlineData(nameof(WriteDoubleSum))]
+        [InlineData(nameof(WriteLongGauge))]
+        [InlineData(nameof(WriteDoubleGauge))]
+        [InlineData(nameof(WriteHistogram))]
+        public void WriteMetricsCollectionTest(string writeActionMethodName)
         {
+            using var meter = new Meter("TestMeter", "0.0.1");
```

Comment:
Consider using a full qualified name so the test cases can run concurrently without introducing conflict.

Reply:
Updated